vignettes/creating-custom-modules.Rmd

```diff
@@ -147,12 +148,12 @@ srv_histogram_example <- function(id, data) {
       plot_code_q()[["p"]]
     })
 
-    # code upon clicking 'Show R Code' button
-    teal.widgets::verbatim_popup_srv(
-      id = "rcode",
-      verbatim_content = reactive(teal.code::get_code(plot_code_q())),
-      title = "R Code"
-    )
+    # view code
+    output$code <- renderPrint({
+      plot_code_q() |>
+        get_code() |>
+        cat()
```

Comment:
`teal` `Depends` on `R >= 4.0` but the pipe is only available in `R 4.1`
```suggestion
      cat(get_code(plot_code_q()))
```

Reply:
Perhaps we can use the magritter pipe here. As there is a `library(teal)` call at the top which would allow us to do this.

Reply:
change to `magrittr` pipe